src/test/test_utils.cpp

```diff
@@ -14,10 +14,73 @@
 // limitations under the License.
 //*****************************************************************************
 #include "test_utils.hpp"
+
+#include "../prediction_service_utils.hpp"
+
+using tensorflow::serving::PredictRequest;
+using tensorflow::serving::PredictResponse;
+
 void waitForOVMSConfigReload(ovms::ModelManager& manager) {
     // This is effectively multiplying by 1.2 to have 1 config reload in between
     // two test steps
-    const float WAIT_MULTIPLIER_FACTOR = 1.2;
+    const float WAIT_MULTIPLIER_FACTOR = 1.1;
```

Comment:
Comment still says 1.2.
Why do we change it to 1.1?

Reply:
Leftover from development - done to have faster tests but at this point it would make sense to rewrite other tests using this function to call proper functions from model manager.cpp. However I think this would be out of scope for this MR.